### PR TITLE
Add forgotten webui_namespace variable

### PR DESCRIPTION
**Pull request by mareklibra**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tools/rotate-certs.sh [hunk 1]
```diff
@@ -22,6 +23,10 @@ while true; do
         shift;
         cdi_namespace=$1
         ;;
+    --webui-namespace)
```
Comment (tiraboschi) [inline tools/rotate-certs.sh]: I think you have also to add `,webui-namespace:` on line 9

Comment (mareklibra) [reply tools/rotate-certs.sh]: Added, thank you. I am currently looking for an environment to run this script altogether, not just by snippets.

**Pull request merged**